jenkins/spark-nightly-build.sh

```diff
@@ -189,6 +190,18 @@ if [[ $SKIP_DEPLOY != 'true' ]]; then
         -Dmaven.scaladoc.skip -Dmaven.scalastyle.skip=true \
         $MVN_URM_MIRROR -Dmaven.repo.local=$M2DIR \
         -Dcuda.version=$DEFAULT_CUDA_CLASSIFIER
+
+    # dist module does not have javadoc and sources jars, use 'sql-plugin' ones instead
+    SQL_ART_PATH="$(echo -n $SQL_PLGUIN_PATH/target/spark*)/rapids-4-spark-sql_${SCALA_BINARY_VER}-${ART_VER}"
```

Comment:
https://github.com/NVIDIA/spark-rapids/blob/branch-25.02/jenkins/deploy.sh#L72-L77
can we share the steps in some common place?

Reply:
We can share the jenkins/deploy.sh for both nightly and sonatype deploy